src/platform/web/lib/http/request/xhrrequest.ts

```diff
@@ -91,7 +74,7 @@ class XHRRequest extends EventEmitter implements IXHRRequest {
     super();
     params = params || {};
     params.rnd = Utils.cheapRandStr();
-    if (needJsonEnvelope() && !params.envelope) params.envelope = 'json';
+    if (!params.envelope) params.envelope = 'json';
```

Comment:
Replace `needJsonEnvelope()` with `false` (which is `needJsonEnvelope()`’s return value in the non-IE10 case)  and you'll see that this is a behaviour change. I think we want to remove this line.

Reply:
Good catch, thank you! Yes, removing the line is the right thing here.